Libplanet.Net.Tests/SwarmTest.AppProtocolVersion.cs

```diff
@@ -152,8 +150,21 @@ AppProtocolVersion localVersion
 
                 foreach (var peer in peers)
                 {
-                    await a.AddPeersAsync(new[] { peer }, null);
-                    await b.AddPeersAsync(new[] { peer }, null);
+                    try
+                    {
+                        await a.AddPeersAsync(new[] { peer }, TimeSpan.FromSeconds(1));
+                    }
+                    catch (PingTimeoutException)
+                    {
+                    }
```

Comment:
eh, is it ok to ignore `PingTimeoutException`?

Reply:
The reason for this is that the behaviors of contacted nodes have changed.

- Before: Nodes replied to *any* message regardless of `APV` signer was trustworthy or not.
- After: Nodes *ignore* messages with an `APV` that is not signed by a trusted source.

Reply:
Then how about use `Assert.Throws<T>()` instead if the exception is always thrown?

Reply:
See 9be99f8d8966fe6438a41348a3bdaa6bec54e3be.